__tests__/__mocks__/zustand.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Source: https://docs.pmnd.rs/zustand/testing (28/03/2022)
+ */
+
+import actualCreate from "zustand";
+import { act } from "react-dom/test-utils";
+import { StoreState } from "@state/useStore";
+
+// a variable to hold reset functions for all stores declared in the app
+const storeResetFns = new Set();
+
+// when creating a store, we get its initial state, create a reset function and add it in the set
+const create = (createState) => {
+	const store = actualCreate<StoreState>(createState);
+	const initialState = store.getState();
+	storeResetFns.add(() => store.setState(initialState, true));
```

Comment:
we don't have multiple stores right now as far as I know so no point in an elaborate setup to support them. KISS